### ISSUE TITLE
Design implementation

### ISSUE THREAD

**Issue by PatriciaKaluza** [wisslab/judaicalink-labs#11]

**Comment by PatriciaKaluza** [comment]
http://www.judaicalink.org/news/experimental-search-online/ 
Habe jetzt angefangen diese Seite (s. Link) mit unserem Mockup-Design zu programmieren (reine HTML & CSS Seite)
-der ganze Code für die Seite steht unter root.html und labs.css
-bis jetzt stehen alle einzelnen Teile (footer, header usw. in einer file)
-der Code ist an vielen Stellen noch nicht perfekt

Das erste Ergebnis sieht so aus:
![erste_HTML_Seite](https://user-images.githubusercontent.com/59728342/76909837-a0396100-68ac-11ea-9150-ce950eb89434.png)

**Comment by PatriciaKaluza** [comment]
Das Hintergrundbild vom Fid habe ich leider nicht, deshalb arbeite ich hier mit einem lizenzfreien Ersatzbild (Quelle: https://pixabay.com/de/vectors/abraham-sarah-jüdische-juden-2147617/).

**Comment by kaiec** [comment]
Das Bild kann man doch runterladen, oder meinen Sie etwas anderes? https://www.ub.uni-frankfurt.de/judaica3/Content-Mgmt/FJS-BG.png

Ansonsten super, können Sie es in Labs einbauen? Ich könnte mich um Hugo kümmern, oder sollen wir das das zusammen via Zoom machen. @marcorovera, wäre das für dich auch interessant?

**Comment by PatriciaKaluza** [comment]
Genau es ging um das Bild danke! 
Wir können es auch dann gleich in Labs einbauen, sollen wir dann auch die Javascript-Funktionen hinzufügen und die Links richtig verknüpfen? 
Auch die Search-Html Seite würden wir dann anfangen. 
Wir können auch gerne via Zoom das weitere Vorgehen besprechen (evtl. am Freitag nach dem Zoom-Meeting für zbw), bis dahin würden wir die Seiten noch perfektionieren (vor allem was die Responsiveness angeht).

**Comment by kaiec** [comment]
Welche Javascript Funktionen? Wir sollten das möglichst modular aufbauen, so dass es verschiedene Basis-Templates gibt, auf denen man aufbauen kann. Die Such-Seite können Sie ebenfalls schon angehen, @marcorovera steht auch bereit, um das von der Backend-Seite mit zu begleiten.
Zoom-Call am Freitag könnte klappen, wenn wir früher fertig werden, aber um 12 habe ich schon eine andere Besprechung. Warten wir es ab, ansonsten können wir da auf jeden Fall kurz das weitere Vorgehen klären.

**Comment by PatriciaKaluza** [comment]
Ach Sie haben Recht, mein Fehler js wird gar nicht benötigt, ich dachte an die Dropdown Navbar.
Dann schauen wir mal ob das am Freitag klappt, ansonsten reagieren Sie ja recht schnell auf offene Fragen in den Issues. :-)

**Comment by PatriciaKaluza** [comment]
https://juedische-studien.fid-lizenzen.de ist das jetzt die neue Seite vom FID? Und wenn ja müssen wir das Design dann wieder daran anpassen, weil das jetzt doch sehr anders aussieht. Hugo habe ich soweit zum laufen gebracht, aber bevor ich da alles reinpacke, wollten wir Labs noch auf einen perfekten Stand bringen, damit nicht alles doppelt angepasst werden muss.

**Comment by kaiec** [comment]
Nein, keine Sorge, das ist eine Seite über alle FIDs von der SUB 
Göttingen.

Viele Grüße,

Kai

On Mon, Mar 30, 2020 at 09:36:46AM -0700, Patricia Kaluza wrote:
>https://juedische-studien.fid-lizenzen.de ist das jetzt die neue Seite vom FID? Und wenn ja müssen wir das Design dann wieder daran anpassen, weil das jetzt doch sehr anders aussieht. Hugo habe ich soweit zum laufen gebracht, aber bevor ich da alles reinpacke, wollten wir Labs noch auf einen perfekten Stand bringen, damit nicht alles doppelt angepasst werden muss.
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/wisslab/judaicalink-labs/issues/11#issuecomment-606107971

-- 
Prof. Dr. Kai Eckert, Stuttgart Media University
http://www.wisslab.org/
PGP Public Key:   http://www.kaiec.org/2012/pgp/pubkey.asc
A987  3760  12A6  35A4  E6D2  577E  513A  6B84  C755  5A67

**Comment by PatriciaKaluza** [comment]
Ok super. Vielen Dank!
